src/risknet/proc/reducer.py

```diff
@@ -31,7 +31,7 @@ def reduce(fm_root, i):
                             'postal_code', 'mortgage_insurance_percent', 'original_loan_term']
 
     df = pd.concat([Reducer.simple_ts_split(pd.read_csv(fm_root + 'historical_data_2009Q1.txt', sep='|', index_col=False,
-                                                            names=origination_cols, nrows=1_000_000).merge(
+                                                            names=origination_cols, nrows=500_000).merge(
```

Comment:
Im a little confused. Why did you change nrows to 500,000?

Reply:
My feature engineering step added more dimensions (features) to the data. I reduced the number of rows because I was worried about using up too much memory!

I plan to test if n=1_000_000 is fine this week.

Reply:
Ohh that makes sense. I tested n=1_000_000 on DSMLP and it works fine. But this is a minor issue. I'll approve the changes so far.